src/main/java/impl/WriterToFileImpl.java

```diff
@@ -0,0 +1,17 @@
+package impl;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import service.WriterToFile;
+
+public class WriterToFileImpl implements WriterToFile {
+    @Override
+    public void writeReportToFile(String report, String toFileName) {
```

Comment:
The method parameter `toFileName` should not be hardcoded inside the service implementation. It is recommended to use a relative path instead, which should be provided from outside of this class, preferably from the Main class.